bin/bazel_to_go.py

```diff
@@ -1,75 +1,200 @@
 #!/usr/bin/env python
 
+#
```

Comment:
I don't think we necessarily need to unify everything. Manager is using gazelle style imports while mixer is not. Mixer is doing funky stuff with proto gen while manager tries to keep it simple. We'll be combining complexity with little benefit to either of us.